pkg/action/failback.go

```diff
@@ -211,4 +216,270 @@ func (ac *ActionController) performDeactivateFailBackStage(action *storkv1.Actio
 			return
 		}
 	}
+	log.ActionLog(action).Infof("Going to update the action status successful for stage %s", storkv1.ActionStageScaleDownDestination)
+	action.Status.Status = storkv1.ActionStatusSuccessful
+	action.Status.Reason = ""
+	ac.updateAction(action)
+}
+
+func (ac *ActionController) performLastMileMigration(action *storkv1.Action) {
```

Comment:
I don't think there will be any difference b/w failover's lastMile and failback's lastMile, right?
Can you please make the required changes so that this method can be directly used by me in failover